conftest.py

```diff
@@ -385,6 +385,11 @@ def serve_wheel(request, tmp_path_factory):
             if existing_url := os.environ.get("PIP_EXTRA_INDEX_URL"):
                 url = f"{existing_url} {url}"
             os.environ["PIP_EXTRA_INDEX_URL"] = url
+            # Set this env var to accelerate uv installation, as http.server
+            # is not able to handle tons of concurrent requests from uv
+            # if we add the url into UV_INDEX environment variable.
+            # used by _create_virtualenv in mlflow.utils.virtualenv
```

Comment:
Is there an option to limit the the number of concurrent requests?

Reply:
Does FastAPI help?

Reply:
Emm I didn't find one

Reply:
Got it, let's try FastAPI.

Reply:
Do we need to? This seems much simpler